content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,38 @@
+---
+title: IMDSv2 Enablement By Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Default Behavior and Host Metadata Enhancement
+
+Agent v7.64.0 enables IMDSv2 by default for enhanced security and improved metadata handling. The host metadata payload contains a field, `legacy-resolution-hostname`, that captures the AWS instance hostname using legacy resolution logic (when IMDSv2 was disabled) and is only included when needed.
+
+The hostname displayed in Datadog **can** change to show the instance ID as the hostname. Datadog continues to use the original hostname to tag the different metrics sent by the Agent.
+
+## Field inclusion criteria
```

Comment:
Given the earlier note about the depth we need, should we remove this section or at least scope it to the visible effects?